service/src/main/java/bio/terra/workspace/service/privateresource/PrivateResourceCleanupService.java

```diff
@@ -162,7 +162,7 @@ private void runCleanupFlight(WorkspaceUserPair workspaceUserPair) {
             .addParameter(WorkspaceFlightMapKeys.ROLE_TO_REMOVE, null);
     try {
       // Skip the job access check as the "manager" role does not grant read access to a workspace.
-      userCleanupJob.submitAndWait(null, /*doAccessCheck=*/ false);
+      userCleanupJob.submitAndWait(null, /*doAccessCheck=*/ false, null);
```

Comment:
nit it's nice to have a comment, similar to the existing `/*doAccessCheck=/` comment. It's hard to tell what `null` means.

```
/* Don't need to update activity log */ null
```